backend/kaggle/datasets_manager.py

```diff
@@ -94,11 +93,35 @@ def __init__(self, dataset_path, full_path=False):
     def get_img_metadata_list(self):
         return self.__img_metadata_list
 
+    def export_img_to_flask(self, index=0, caption_style='HAM'):
```

Comment:
I think you can remove `get_img_metadata_list` and rename this to `get_img_metadata`

Reply:
@dinhanhx should've replied to this but ok.

What I mean is delete that useless function because it's not going to be called anyway, and rename `export_img_to_flask` to `get_img_metadata`. Are you clear?

Reply:
Oh just realized you were replying to my question in the big comment.

Reply:
> @dinhanhx should've replied to this but ok.
> 
> What I mean is delete that useless function because it's not going to be called anyway, and rename `export_img_to_flask` to `get_img_metadata`. Are you clear?

Yeah I got your point.